inductiva/tasks/methods.py

```diff
@@ -49,3 +50,45 @@ def fetch_task_output(task_id: str,
             result_list = api.download_output(api_instance, task_id, output_dir)
 
             return unpack_output(result_list, output_dir, return_type)
+
+
+def get_task(task_id) -> Dict:
+    api_config = api.validate_api_key(inductiva.api_key)
+
+    with ApiClient(api_config) as client:
+        api_instance = TasksApi(client)
+
+        try:
+            # Get User Tasks
+            api_response = api_instance.get_task(
+                path_params={"task_id": task_id})
+
+            return {**api_response.body}
+
+        except ApiException as e:
+            raise e
+
+
+def get_tasks(status: Optional[str] = None, page=1, per_page=10) -> List[Dict]:
+    api_config = api.validate_api_key(inductiva.api_key)
+
+    with ApiClient(api_config) as client:
+        api_instance = TasksApi(client)
+
+        query_params = {
```

Comment:
try to avoid code duplication regarding query_params, perhaps using that namedtuple I suggested... not sure..